src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (Object.keys(inverse).includes(value.toString())) {
+      return null;
+    } else {
+      inverse[value] = key;
+    }
```

Comment:
```suggestion
    } 
    
    inverse[value] = key;
```
try to avoid else where it is possible